src/main/webapp/signup.js

```diff
@@ -116,21 +113,7 @@ function createSignupRequest() {
     encodedBody.append('interestTags', interestTag);
   });
 
-  const fetchRequest = fetch(makeRelativeUrlAbsolute('/user'), {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/x-www-form-urlencoded',
-    },
-    body: encodedBody,
-  });
-
-  const errorFormattedFetchRequest = standardizeFetchErrors(
-      fetchRequest,
-      'Failed to communicate with the server. Please try again later.',
-      'An error occcured while creating your account.' +
-          ' Please try again later.');
-
-  return errorFormattedFetchRequest;
+  return postUser(encodedBody);
```

Comment:
Is there a reason why this code was moved? Will there ever be another use beyond signup where a user will need to be posted to the `/users` servlet? If your intention here is to make the post functionality more generally accessible if we intend to allow users to edit their own profiles in the future, consider following [RESTful API standards](https://www.restapitutorial.com/lessons/httpmethods.html#:~:text=The%20primary%20or%20most%2Dcommonly,but%20are%20utilized%20less%20frequently.) and using a [`doPut()`](https://tomcat.apache.org/tomcat-5.5-doc/servletapi/javax/servlet/http/HttpServlet.html#doPut(javax.servlet.http.HttpServletRequest,%20javax.servlet.http.HttpServletResponse)) and sending a PUT request in that case. That will allow us to keep the two functionalities separate and follow proper standards.

Reply:
Oh good catch! I'll use the ```doPut()```. My reason for moving the function was to make it more accessible but I guess it won't really be used anywhere else. That being said, I'd still like to have it in the ```user.js``` module. I like having all the functions that use the ```/user``` servlet in the same place. If you think it's better in another module, that's fine too

Reply:
As long as we're on the same page that it really will only be used for signup, I'd prefer if it stayed in the original file `signup.js`. That way we can avoid confusion with the post request vs. put request and we can have more freedom with changes to this function down the line. Also, when you moved it, you exposed a lot of the error checks and content limiting. Before one was only able to make a post request with specific signup body data, and now a post request can be made with any URLEncoded data.